### PR TITLE
Financial documentation updated

### PR DESCRIPTION
**Pull request by raipc**

### CODE REVIEW

File: finance/command-statistics-insert.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ The event time is `2021-01-14T11:02:50.591005Z`
 
 * New instruments are automatically registered as entities with name `<symbol>_[<class>]`, for example `gazp_[tqbr]` for class `TQBR` and symbol `GAZP`.
 
-* If the value of `microseconds` field exceeds 1000, it is added to milliseconds. `1588283343000,645713` is the same as `1588283343645,713`.
+* If the number of digits in `fractions` field exceeds 3, it is treated as nanoseconds, microseconds otherwise. `0003` will count as 3 nanoseconds, while `3` as 3000 nanoseconds.
```
Comment (rodionos) [inline finance/command-statistics-insert.md:53]: Is it how it actually is processed? I thought we interpret `3000` as 3 milliseconds. This was due to support quirks in the QUIK exporter.

File: finance/overview-ru.md [hunk 2]
```diff
@@ -4,9 +4,9 @@
 
 ## Интеграция с Московской Биржей
 
-ATSD оптимизирована для обработки биржевой информации Московской Биржи по высоко-производительному протоколу [FAST Multicast](https://www.moex.com/s441) в режиме реального времени.
+ATSD оптимизирована для обработки биржевой информации Московской Биржи по высокопроизводительному протоколу [FAST Multicast](https://www.moex.com/s441) в режиме реального времени.
 
-* Получение данных с Фондового, Денежного, Валютного, и Срочного рынков
+* Получение данных с Фондового, Валютного, и Срочного рынков
```
Comment (rodionos) [inline finance/overview-ru.md:9]: Денежный рынок - это классы РЕПО, но можно оставить "Фондового, Валютного, и Срочного"/

File: finance/command-statistics-insert.md (hunk 1 above)
Comment (raipc) [reply finance/command-statistics-insert.md:53]: Such trick is used in trades endpoint. Statistics endpoint is different: it is used by moex-consumer only. ASTS supports microsecond precision, Spectra keeps timestamps in nanoseconds. At first we had only ASTS consumer implemented. Digits count-based precision determination allows to keep backward compatibility between ASTS and Spectra statistics, and compression inefficiency will be touched by only 0.1% of statistics in Spectra

File: finance/overview-ru.md (hunk 2 above)
Comment (raipc) [reply finance/overview-ru.md:9]: Действительно, в ASTS присутствуют данные денежного рынка. Откачу это изменение

**Pull request merged**